docs/source/distributions.rst

```diff
@@ -91,6 +91,12 @@ Probability distributions - torch.distributions
 .. autoclass:: Laplace
     :members:
 
+:hidden:`MultivariateNormal`
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
nit: underline is too short and will break docs. You can run `make -C docs html` and open `docs/build/html/index.html` to check docs.